src/utils/ADempiere/valueUtils.js

```diff
@@ -420,6 +420,18 @@ export function tagStatus(tag) {
   }
   return type
 }
+export function formatDateConvert(date) {
```

Comment:
What is the reason of this function if already exists a function for it. Please don't rounded the code if already exists